trl/trainer/dpo_config.py

```diff
@@ -156,11 +157,9 @@ class DPOConfig(TrainingArguments):
             [MPO](https://huggingface.co/papers/2411.10442)). The `loss_weights` parameter can be used to specify
             corresponding weights for each loss type.
 
-        use_liger_loss (`bool`, *optional*, defaults to `False`):
-            Whether to use Liger loss.
```

Comment:
I think we should keep the parameter in the documentation as deprecated, until it is effectively removed:
```python
        use_liger_loss (`bool`, *optional*, defaults to `False`):
            Whether to use Liger loss.

            <Deprecated version="0.25.0">

            Parameter `use_liger_loss` is deprecated and will be removed in version 0.28.0. Use `use_liger_kernel` instead.

            </Deprecated>
```